internal/view/actions.go

```diff
@@ -107,12 +109,12 @@ func pluginActions(r Runner, aa ui.KeyActions) {
 		}
 		aa[key] = ui.NewKeyAction(
 			plugin.Description,
-			execCmd(r, plugin.Command, plugin.Background, plugin.Args...),
+			execCmd(r, plugin.Command, plugin.Background, plugin.ConfirmDialog, plugin.Args...),
```

Comment:
Let change execCmd to something like pluginAction as it's only used in this context. Also might as well pass in the config.Plugin ie pluginAction(r, plugin) which returns an ActionHandler.